clients/nutclient.cpp

```diff
@@ -730,7 +730,7 @@ std::map<std::string,std::map<std::string,std::vector<std::string> > > TcpClient
 			{
 				std::vector<std::string>& vals = *it2;
 				std::string var = vals[0];
-				vals.erase(vals.begin()),
+				vals.erase(vals.begin());
```

Comment:
@aquette @boricj : please tell me I did fix a typo and not some intentional C++ trick? ;)

Reply:
I'd say harmless typo.